javascript/soyutils_velog.js

```diff
@@ -29,11 +29,25 @@ goog.module('soy.velog');
 goog.module.declareLegacyNamespace();
 
 const Message = goog.require('jspb.Message');
-const xid = goog.require('xid');
 const {assert} = goog.require('goog.asserts');
 const {getFirstElementChild, getNextElementSibling} = goog.require('goog.dom');
 const {startsWith} = goog.require('goog.string');
 
+
+/**
+ * ID generator for soy operations.
+ *
+ * @idGenerator {xid}
+ * @param {!string} id Un-rewritten ID to use.
+ * @return {!string} ID to use in code (potentially rewritten).
+ */
+function xidGen(id) {
+  return id;
```

Comment:
basically i installed a simple `xidGen` function, which works in `@idGenerator {xid}` mode.